microsoft-support.md

```diff
@@ -0,0 +1,37 @@
+# Microsoft support for grpc-dotnet
```

Comment:
I think we need a paragraphs to set the context about the overall situation with grpc-dotnet support.
I want to make it clear that grpc-dotnet is a CNCF project, not a Microsoft project and that microsoft support is basically "a bonus" on top of the normal support grpc-dotnet gets.

Can we start the document with something like:
 _"gRPC is a CNCF project and receives support from the gRPC team and the grpc-dotnet maintainers and contributors. (same situation as all the other officially supported gRPC languages). Besides that, since grpc-dotnet is widely recognized as an important technology for the .NET ecosystem, it also receives official support directly from Microsoft's support team. This document summarized the details of Microsoft's support policy: ...."_

The exact wording is not important, but the overall message is important to convey IMHO.

Reply:
Done.